src/components/custom-tom-select.scss

```diff
@@ -292,7 +276,7 @@
   top: 100%;
   left: 0;
   width: 100%;
-  z-index: 10;
+  z-index: var(--rms-z-index-dropdown);
   border: 1px solid #d0d0d0;
```

Comment:
@Jeremy-Walton Can we use our tokens here instead of the static styles for border color.